packages/utils/package-manager.ts

```diff
@@ -13,17 +13,16 @@ import * as path from "path";
  */
 
 export function getPackageManager(): string {
-	const hasLocalNPM = fs.existsSync(path.resolve(process.cwd(), "package-lock.json"));
-	const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
-	if (hasLocalNPM) {
-		return "npm";
-	} else if (hasLocalYarn) {
-		return "yarn";
-	} else if (spawn.sync("yarn", [" --version"], { stdio: "ignore" }).error) {
-		return "npm";
-	} else {
-		return "npm";
-	}
+    const pwd = path.resolve(process.cwd());
+    const hasLocalYarn = fs.existsSync(path.resolve(pwd + '/yarn.lock'));
+
+    if (hasLocalYarn) {
+        return 'yarn';
+    } else if (spawn.sync('yarn', [' --version'], { stdio: 'ignore' }).error) {
```

Comment:
Could you do a early return instead and do a if/return statement?

Reply:
I didn't get it, could you pls elaborate.

Reply:
It means
```js
if () {
   return
}
return;
```
No need of `else` statement.
Also, here you forgot to kill the process.

Reply:
Also, could you use `execa` instead?